File: C#/Ejercicio7.cs
using System;

void SelectionSort(int[] arr)
{
    int n = arr.Length;
    for (int i = 0; i < n; i++) {
        int minIndex = i;
        for (int j = i + 1; j < n; j++) {
            if (arr[j] < arr[minIndex]) {
                minIndex = j;
            }
        }
        int actual = arr[i];
        arr[i] = arr[minIndex];
        arr[minIndex] = actual;
    }
}

int[] arr = {64, 34, 25, 12, 22, 11, 90};
Console.WriteLine("=== SELECTION SORT EN C# SCRIPT ===");
Console.WriteLine("Array inicial: [" + string.Join(", ", arr) + "]");

SelectionSort(arr);

Console.WriteLine("Array final: [" + string.Join(", ", arr) + "]");

File: Python/Ejercicio7.py
def selectionSort(arr):
    for i in range(len(arr)):
        min_index = i
        for j in range(i + 1, len(arr)):
            if arr[j] < arr[min_index]:
                min_index = j
        arr[i], arr[min_index] = arr[min_index], arr[i]

def main():
    arr = [32, 17, 12, 6, 11, 5, 45]

    print("=== SELECTION SORT EN PYTHON ===")
    print(f"Array inicial: {arr}")

    selectionSort(arr.copy())

    arrSorted = arr.copy()
    selectionSort(arrSorted)
    print(f"Array final: {arrSorted}")

if __name__ == "__main__":
    main()